config/routes.rb

```diff
@@ -202,6 +201,7 @@
 
     resource :session, only: %i[create destroy] do
       post 'mfa_create', to: 'sessions#mfa_create', as: :mfa_create
+      post 'webauthn_create', to: 'sessions#webauthn_create', as: :webauthn_create
```

Comment:
Moving `webauthn_create` to the UI block since its response is constrained to html instead of json.